src/i18n/i18n.translations.ts

```diff
@@ -0,0 +1,49 @@
+import { namespaces } from "./i18n.constants";
+
+export const en = {
+    [namespaces.common]: {
+      buttons: {
+        ok: "Ok",
+        cancel: "Cancel",
+      },
+    },
+    [namespaces.pages.hello]: {
+      welcome: "Welcome to Cultural Survival.",
+      categories: "Categories",
+      radio: "Radio",
+      grants: "Grants",
+      
+    },
+    [namespaces.pages.grants]: {
+      grants_feed: "Grants Available",
+      
+    },
+    [namespaces.pages.audio]: {
+      audio_feed: "AudioFEED_Feed",
+      
+    },
+  };
+
+export const es = {
+  [namespaces.common]: {
+    buttons: {
+      ok: "Aceptar",
+      cancel: "Cancelar",
+    },
+  },
+  [namespaces.pages.hello]: {
+    welcome: "Bienvenida a Cultural Survival.",
+    categories: "Categorías",
+    radio: "Radio",
+    grants: "Grants Feed",
+  
+  },
+  [namespaces.pages.grants]: {
+    grants_feed: "Subvenciones Disponibles",
+  },
+  [namespaces.pages.audio]: {
+    audio_feed: "AudioFIREBASE_Alimento",
```

Comment:
"AudioFIREBASE_Alimento" is not a direct translation of "AudioFEED_feed". The correct translation is "Alimentación de Audio".